ci/conda_env_unix.txt

```diff
@@ -19,5 +19,7 @@
 
 autoconf
 ccache
+google-cloud-cpp=1.20.0
+nlohmann_json>=3.4.0
```

Comment:
what is the new JSON dependency for?

Reply:
I guess google's SDK must rely on it?

Reply:
I am one of the authors of google's GCS SDK.  Yes, the SDK definitely depends on `nlohmann_json`.

Reply:
Conda has dependency handling, so there is no reason to mention it explicitly, though.